src/pip/_internal/vcs/subversion.py

```diff
@@ -31,34 +31,6 @@ class Subversion(VersionControl):
     def get_base_rev_args(self, rev):
         return ['-r', rev]
 
-    def get_info(self, location):
```

Comment:
For getting the URL, I wonder which is better, the logic being deleted here or the logic in `get_url()`? This logic looks a bit simpler FWIW. It's curious that it's different because they're both supposed to do the same thing.

Reply:
`get_url()` relies on `_get_svn_url_rev()` to do its dirty work. I think `_get_svn_url_rev()` is more complicated than `get_info()` because it attempts to read the URL from disk before falling back to spawning a `svn info` child process (which is all `get_info()` does).

Reply:
That was more or less my assessment, too. I wonder if we should consider simplifying svn's `get_url()` implementation with something more like `get_info()` (but not for this PR, of course). `get_info()` is simpler because it relies only on svn's command API inside of what looks like parsing svn implementation details that seem to vary from version to version.

Reply:
I think the intention is that reading from disk is orders of magnitude faster than launching a child process, otherwise the original author wouldn't have bothered!

Reply:
It’s certainly possible. But without seeing a code comment, issue discussion, or asking the author, I’m not sure we can say for sure.

Reply:
> and I'd rather withdraw this PR than be forced to make it even slower for the sake of removing a vestigial method.

I wasn't suggesting that you change your PR. I said, "I wonder if we should consider simplifying svn's `get_url()` implementation _in the future_..." and "not for this PR, of course." It was an observation / idea. I approved your PR.